### ISSUE TITLE
Update ScummVM games

### ISSUE THREAD

**Issue by 3XistencE** [libretro/libretro-database#1007]
Goodmorning,
actually a lot of scummmVM games are not inside the database (it derivates that the final user can't scan them into playlist even with a proper .scummvm file on the directory and make RA crash on desktop if imported and launched with "scan files manually").
I suggest to update the database list in order to make the games runnable from playlist without experiencing a RA crash to desktop

I was able to test only few of them but Most of Sierra SCI and AGI (present as workaround into the .dat file as "Sierra AGI game" "Sierra SCI game" without specifications.
Tested with RA crash if imported manually
+King's Quest I (SCI remake) | kq1sci
+Quest for Glory 1 VGA remake | qfg1vga
+Space Quest I (SCI remake) (EGA and VGA) | sq1sci
+Police Quest 1 (SCI remake) | pq1sci
tried (with the same problem)
+The Lost Files of Sherlock Holmes: The Case of the Serrated Scalpel | scalpel

I experienced also some problems with Gabriel Knight: Sins of the Fathers but managed to make it work

**Comment by RobLoach** [comment]
Had a look at indexing the game files instead....
https://github.com/libretro/libretro-database/pull/1000

Curious what your thoughts are on that approach.

**Comment by 3XistencE** [comment]
For sure is a better idea, the final user will not need to create a .scummvm file.
Can i have a direct link to the file in order to check the code and, if i will be able, to help with indexing?

**Comment by RobLoach** [comment]
The ScummVM autolaunching has been merged, along with the DAT being rebuilt. It may take a while for it all to get on buildbot, but would love your help in testing all this out!

1. Update database
2. Update the ScummVM core
3. Delete/backup your .scummvm files
3. Try scanning some games without the .scummvm files :wink:

**Comment by 3XistencE** [comment]
> The ScummVM autolaunching has been merged, along with the DAT being rebuilt. It may take a while for it all to get on buildbot, but would love your help in testing all this out!
> 
> 1. Update database
> 2. Update the ScummVM core
> 3. Delete/backup your .scummvm files
> 4. Try scanning some games without the .scummvm files wink

Hi Rob,
I will update all retroarch+core+databases and test without .scummvm if are recognized by the databases, in case of missing one I noticed that most of the work is based on single files+crc32 (probably based on scummvm dat files documentation). In that case I can write here down file.ext +crc and complete name+version+language in order to add them to database if it's helpful.

Sorry to write it here but somebody can check also the gme core, I have issues and post it but nobody actually watched it (the same for openlara and reminiscence cores).
For openlara maybe it's enough to update the core with the openlara project git code, for the others I can't figure out the problem.

Thank You

**Comment by RobLoach** [comment]
> I can write here down file.ext +crc and complete name+version+language in order to add them to database if it's helpful.

That would be very helpful. There were cases that could be missed so we could add them in manually over at the building script:
https://github.com/robloach/libretro-database-scummvm

> Sorry to write it here but somebody can check also the gme core,

I don't know who's actively maintaining that one. Your best bet is to do as you did and open up an issue there.

**Comment by RobLoach** [comment]
The game files is in. You shouldn't need the .scummvm files anymore, as they are scanned in with the CRCs. There are a few that have duplicate files, and couldn't find a good file to index:
https://github.com/RobLoach/libretro-database-scummvm/blob/master/missing-games.txt

We may have to manually fix those ones.

Going to close this one. OpenLara and Reminscence repos could do with more updates. Feel free to open up issues there.

**Comment by lasers** [comment]
>Going to close this one.

Please revisit this issue. I'm not sure where we stand here today. Thank you.

**Comment by Magicrafter13** [comment]
> Had a look at indexing the game files instead....
> #1000
> 
> Curious what your thoughts are on that approach. Then you wouldn't need the .scummvm files at all.

By any chance, is this why many (if not all) games are repeated several times?
![browsing-database](https://user-images.githubusercontent.com/29127566/128615889-8f395878-475c-493f-96c3-c0aac9e9f18e.png)

**Comment by RobLoach** [comment]
Correct, those list all the different languages, along with their .scummvm files with newlines. Could probably restore the language name in the parameters, along with removing the entries with the newline at the end of the .scummvm files.
